service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEventHandlerTest.java

```diff
@@ -42,17 +46,42 @@
 import static uk.gov.hmcts.reform.fpl.handlers.FurtherEvidenceUploadedEventTestData.userDetailsHMCTS;
 import static uk.gov.hmcts.reform.fpl.handlers.FurtherEvidenceUploadedEventTestData.userDetailsLA;
 import static uk.gov.hmcts.reform.fpl.handlers.FurtherEvidenceUploadedEventTestData.userDetailsRespondentSolicitor;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
 import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
 
 @ExtendWith(MockitoExtension.class)
-class FurtherEvidenceUploadedEventHandlerSendNotificationTest {
-
+class FurtherEvidenceUploadedEventHandlerTest {
+    private static final Long CASE_ID = 12345L;
+    private static final String CONFIDENTIAL_MARKER = "CONFIDENTIAL";
+    private static final String LA_USER = "LA";
+    private static final String HMCTS_USER = "HMCTS";
+    private static final String REP_USER = "REP";
+    private static final String LA_USER_EMAIL = "la@examaple.com";
+    private static final String HMCTS_USER_EMAIL = "hmcts@examaple.com";
+    private static final String REP_SOLICITOR_USER_EMAIL = "rep@examaple.com";
+    private static final String SENDER_FORENAME = "The";
+    private static final String SENDER_SURNAME = "Sender";
+    private static final String SENDER = SENDER_FORENAME + " " + SENDER_SURNAME;
+    private static final String REP_SOLICITOR_1_EMAIL = "rep_solicitor1@example.com";
+    private static final String REP_SOLICITOR_2_EMAIL = "rep_solicitor2@example.com";
+    private static final LocalDateTime HEARING_DATE = LocalDateTime.now().plusMonths(3);
+    private static final CaseData CASE_DATA = mock(CaseData.class);
```

Comment:
need to be careful about static mocks here. I know that @tomaszpowroznik had to change a few instances of using static case data to instanced in other tests

Reply:
a constant should never be instructed with WHEN()